packages/playwright/src/reporters/github.ts

```diff
@@ -56,9 +56,14 @@ class GitHubLogger {
   }
 }
 
-export class GitHubReporter extends BaseReporter {
+export class GitHubReporter extends TerminalReporter {
   githubLogger = new GitHubLogger();
 
+  constructor(options: { omitFailures?: boolean } = {}) {
+    super(options);
+    this.screen.colors = noColors;
```

Comment:
To avoid changing the global `terminalScreen.colors` and messing up other reporters:

```suggestion
    this.screen = { ...this.screen, colors: noColors };
```

Reply:
Nice catch